ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -48,6 +49,7 @@ const getInitialFormState = (
     (acc, field) => ({
       ...acc,
       [field.id]: {
+        ...field.validation,
```

Comment:
Are we expecting the validation property to always exist or be itterable? Do we need a fallback/short circuit here?